src/config.rs

```diff
@@ -128,6 +150,29 @@ impl Context {
         self.get_config_int(key) != 0
     }
 
+    /// Gets configured "delete_server_after" value.
+    ///
+    /// `None` means never delete the message, `Some(0)` means delete
+    /// at once, `Some(x)` means delete after `x` seconds.
+    pub fn get_config_delete_server_after(&self) -> Option<i64> {
```

Comment:
Isn't std::time::Duration more suitable than i64 here?  (but also note the comments on integer conversions below)